doc/whats_new/v1.3.rst

```diff
@@ -186,6 +186,11 @@ Changelog
   dissimilarity is not a metric and cannot be supported by the BallTree.
   :pr:`25417` by :user:`Guillaume Lemaitre <glemaitre>`.
 
+- |Enhancement| The performance of :meth:`neighbors.KNeighborsClassifier.predict`
+  and of :meth:`neighbors.KNeighborsClassifier.predict_proba` has been improved
+  for all distance metrics.
+  :pr:`24076` by :user:`Meekail Zain <micky774>`, :user:`Julien Jerphanion <jjerphan>`.
```

Comment:
Huh, I've always considered reviewers as essentially co-authors regardless. I guess the sentiment is a bit different from what I had understood? I'm fine with it either way, I just enjoy listing people when their contributions in the review phase include significant changes in code

Reply:
I would wait for @ogrisel's opinion.

Reply:
Historically, we sometimes included people besides the original authors in the change log entry, which was done case by case. Overall, I'm okay with having a policy of "Include everyone that contributed significant amount of code". This would include code that was contributed during the review process.